web/src/components/ConsentBanner.test.js

```diff
@@ -2,14 +2,13 @@ import { shallow } from 'enzyme';
 import React from 'react';
 import sinon from 'sinon';
 
+import ConsentBanner from './ConsentBanner';
+
 describe('Consent banner component', () => {
-  let ConsentBanner;
   const onAgree = sinon.spy();
 
   beforeEach(() => {
-    jest.resetModules();
     onAgree.resetHistory();
-    ConsentBanner = require('./ConsentBanner').default; // eslint-disable-line global-require
```

Comment:
All this module reloading stuff can go away now.

Reply:
![aw yeah](https://media0.giphy.com/media/oDnDT2PnCZjTa/source.gif)